.github/workflows/hil_unity_checks.yml

```diff
@@ -134,11 +135,11 @@ jobs:
           rm -rf $HOME/artefact
           mkdir -p $HOME/artefact
           
-          cd ~/.arduino15/packages/infineon/hardware/psoc/$TEST_VERSION/tests/unity/
+          cd ~/.arduino15/packages/infineon/hardware/psoc/$TEST_VERSION/tests/arduino-core-tests/Unity/
           echo "=============================================================================================================="
           echo "Run Tests"
           echo "=============================================================================================================="
-          make FQBN=${{ env.fqbn }} PORT=/dev/ttyACM0 single_hardware_connected
+          make FQBN=${{ env.fqbn }} PORT=/dev/ttyACM0 math
```

Comment:
The "math" tests are not present anymore?

Reply:
yes. I didn't rework this since it's already been done as part of the HIL setup ticket for multiple boards